OSPSuite.Presentation.Importer/Core/IDataSourceFile.cs

```diff
@@ -0,0 +1,40 @@
+﻿using OSPSuite.Infrastructure.Import.Services;
+using System.Collections.Generic;
+
+namespace OSPSuite.Presentation.Importer.Core
+{
+   /// <summary>
+   /// Single file containing the data, e.g. excel file or csv file
+   /// </summary>
+   public interface IDataSourceFile
+   {
+      Dictionary<string, IDataSheet> DataSheets { get; set; }
```

Comment:
No get here. 
I think when definining contracts it makes sense to decide what is readonly and what not
Default as everything {get;set} makes it hard for the 'outsider' to guess how a entity is to be used